cli/src/config.ts

```diff
@@ -237,6 +237,8 @@ async function loadIOSConfig(
   const platformDirAbs = resolve(rootDir, platformDir);
   const nativeProjectDir = 'App';
   const nativeTargetDir = `${nativeProjectDir}/App`;
+  const nativeXcodeProjDir = `${nativeProjectDir}/App.xcodeproj`;
+  const nativeXcodeWorkspaceDir = `${nativeProjectDir}/App.xcworkspace`;
```

Comment:
Note: with this PR, we now blindly assume `App.xcworkspace` (instead of what was happening in `findXcodePath()`).

What was the reason for `findXcodePath()`? Are there apps with workspace directories named something other than `App.xcworkspace`?

Reply:
It's not possible at the moment since we hardcode `App` in a few places, but users have tried to rename their projects and it broke because our hardcoded values.